c7n/resources/appelb.py

```diff
@@ -760,6 +761,26 @@ def process(self, load_balancers):
                     **args)
 
 
+@AppELB.action_registry.register('modify-security-groups')
+class AppELBModifyVpcSecurityGroups(ModifyVpcSecurityGroupsAction):
+
+    permissions = ("elasticloadbalancing:SetSecurityGroups",)
+
+    def process(self, albs):
+        client = local_session(self.manager.session_factory).client('elbv2')
+        groups = super(AppELBModifyVpcSecurityGroups, self).get_groups(albs)
+
+        for idx, i in enumerate(albs):
+            try:
+                client.set_security_groups(
+                    LoadBalancerArn=i['LoadBalancerArn'],
+                    SecurityGroups=groups[idx])
+            except ClientError as e:
```

Comment:
this can be simplified by using the exception attributes on the client ie.

```python
except client.exceptions.LoadBalancerNotFoundException:
     continue
```

Reply:
thanks for this comment, helped me resolve the other PR as well. coverage looks good for both now!